webui/react/src/pages/ExperimentDetails/ExperimentDetailsHeader.tsx

```diff
@@ -371,6 +370,7 @@ const ExperimentDetailsHeader: React.FC<Props> = ({
                 isOnDark
                 maxLength={128}
                 placeholder="experiment name"
+                showSpinner={false}
```

Comment:
we still want spinner to appear when it's saving, we just don't want it to change the screen layout.